src/components/Settings/sections/ModelManagementSection.scss

```
@@ -251,6 +198,10 @@
     display: flex;
     align-items: center;
     gap: 8px;
+    // The variant dropdown's row height, so a card with the dropdown and a
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)